Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.yml

```
@@ -0,0 +1,1891 @@
+      description: Negate the destination.
+      isArray: false
+      name: negate_destination
+      required: false
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    description: Create a new security rule.
+    execution: false
+    name: prisma-sase-security-rule-create
+    outputs:
+    - contextPath: PrismaSase.SecurityRule
+      description: Created security rule.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.action
+      description: Security rule action.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.application
+      description: Security rule application.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.category
+      description: Security rule category.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.description
+      description: Security rule description.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.destination
+      description: Security rule destination.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.folder
+      description: Security rule folder.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.from
+      description: Security rule from field (source zone(s)).
+      type: String
+    - contextPath: PrismaSase.SecurityRule.id
+      description: Security rule ID.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.name
+      description: Security rule name.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.position
+      description: Security rule position.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.profile_setting.group
+      description: Security rule group.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.service
+      description: Security rule service.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.source
+      description: Security rule source.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.source_user
+      description: Security rule source user.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.to
+      description: Security rule to field (destination zone(s)).
+      type: String
+  - arguments:
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
+      isArray: false
+      name: folder
+      predefined:
+      - Shared
+      - Mobile Users
+      - Remote Networks
+      - Service Connections
+      - Mobile Users Container
+      - Mobile Users Explicit Proxy
+      required: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: pre
+      description: Security rule position.
+      isArray: false
+      name: position
+      predefined:
+      - pre
+      - post
+    - defaultValue: '50'
+      description: The maximum number of results to return. Default is 50.
```

```suggestion
      description: The maximum number of results to return. 
```
No need to put the default value in the description.  When the README is generated it automatically picks up the default value from defaultValue.